src/impl_methods.rs

```diff
@@ -732,14 +732,28 @@ where
     /// ```
     pub fn get<I>(&self, index: I) -> Option<&A>
     where
-        I: NdIndex<D>,
         S: Data,
+        I: NdIndex<D>,
     {
         unsafe { self.get_ptr(index).map(|ptr| &*ptr) }
     }
 
-    pub(crate) fn get_ptr<I>(&self, index: I) -> Option<*const A>
+    /// Return a raw pointer to the element at `index`, or return `None`
+    /// if the index is out of bounds.
+    ///
+    /// ```
+    /// use ndarray::arr2;
+    ///
+    /// let a = arr2(&[[1., 2.], [3., 4.]]);
+    ///
+    /// let v = a.raw_view();
+    /// let p = a.get_ptr((0, 1)).unwrap();
+    ///
+    /// assert_eq!(unsafe { *p }, 2.);
+    /// ```
+    pub fn get_ptr<I>(&self, index: I) -> Option<*const A>
     where
+        S: RawData,
```

Comment:
Huh, that's weird. I don't know why this didn't show up on the review.

This line isn't necessary, since the `impl` block already has a `S: RawData` bound.

```suggestion
```

Reply:
Fixed.